lib/cocoapods-core/specification.rb

```diff
@@ -74,6 +79,11 @@ def initialize_copy(other)
     attr_accessor :test_specification
     alias_method :test_specification?, :test_specification
 
+    # @return [Bool] If this specification is an app specification.
+    #
+    attr_accessor :app_specification
```

Comment:
I was stupid when I did test specs and this should probably be `attr_reader`

Reply:
These actually need to be accessor because of the way we convert to and from json